buildSrc/src/main/kotlin/Deps.kt

```diff
@@ -33,6 +33,13 @@ object deps {
         const val runtime = "com.squareup.sqldelight:runtime:${Versions.sqldelight}"
     }
 
+    object ktor {
+        const val androidKdriver = "io.ktor:ktor-client-okhttp:${Versions.ktor}"
+        const val iosKdriver = "io.ktor:ktor-client-ios:${Versions.ktor}"
+        const val jsKdriver = "io.ktor:ktor-client-js:${Versions.ktor}"
+     // const val commonKdriver = "io.ktor:ktor-client-core:${Versions.ktor}"
```

Comment:
```suggestion
        const val androidDriver = "io.ktor:ktor-client-okhttp:${Versions.ktor}"
        const val iosDriver = "io.ktor:ktor-client-ios:${Versions.ktor}"
        const val jsDriver = "io.ktor:ktor-client-js:${Versions.ktor}"
     // const val commonDriver = "io.ktor:ktor-client-core:${Versions.ktor}"
```

We can keep the names simple